[2023-05-19T00:39:50.301+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: combined_dag.submit_storm_topology_task scheduled__2023-05-18T00:06:00+00:00 [queued]>
[2023-05-19T00:39:50.311+0530] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: combined_dag.submit_storm_topology_task scheduled__2023-05-18T00:06:00+00:00 [queued]>
[2023-05-19T00:39:50.311+0530] {taskinstance.py:1331} INFO - Starting attempt 1 of 4
[2023-05-19T00:39:50.327+0530] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): submit_storm_topology_task> on 2023-05-18 00:06:00+00:00
[2023-05-19T00:39:50.331+0530] {standard_task_runner.py:57} INFO - Started process 10484 to run task
[2023-05-19T00:39:50.335+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'combined_dag', 'submit_storm_topology_task', 'scheduled__2023-05-18T00:06:00+00:00', '--job-id', '532', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpdkghf1v6']
[2023-05-19T00:39:50.336+0530] {standard_task_runner.py:85} INFO - Job 532: Subtask submit_storm_topology_task
[2023-05-19T00:39:50.383+0530] {task_command.py:410} INFO - Running <TaskInstance: combined_dag.submit_storm_topology_task scheduled__2023-05-18T00:06:00+00:00 [running]> on host nizam-HP-15-Notebook-PC
[2023-05-19T00:39:50.461+0530] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='combined_dag' AIRFLOW_CTX_TASK_ID='submit_storm_topology_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-18T00:06:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-18T00:06:00+00:00'
[2023-05-19T00:40:03.425+0530] {logging_mixin.py:149} INFO - Cleaning from prior builds...
[2023-05-19T00:40:05.365+0530] {logging_mixin.py:149} INFO - Creating topology Uber-JAR...
[2023-05-19T00:40:10.977+0530] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 10484. PIDs of all processes in the group: [10485, 10486, 10621, 10622, 10629, 10484]
[2023-05-19T00:40:10.978+0530] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 10484
[2023-05-19T00:40:10.978+0530] {taskinstance.py:1540} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-05-19T00:40:10.979+0530] {logging_mixin.py:149} INFO - Error occurred in submit_topology: Task received SIGTERM signal
[2023-05-19T00:40:10.980+0530] {python.py:183} INFO - Done. Returned value was: None
[2023-05-19T00:40:10.989+0530] {process_utils.py:79} INFO - Process psutil.Process(pid=10621, status='terminated', started='00:40:04') (10621) terminated with exit code None
[2023-05-19T00:40:10.990+0530] {process_utils.py:79} INFO - Process psutil.Process(pid=10485, status='terminated', started='00:39:49') (10485) terminated with exit code None
[2023-05-19T00:40:10.990+0530] {process_utils.py:79} INFO - Process psutil.Process(pid=10486, status='terminated', started='00:39:49') (10486) terminated with exit code None
[2023-05-19T00:40:10.991+0530] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=combined_dag, task_id=submit_storm_topology_task, execution_date=20230518T000600, start_date=20230518T190950, end_date=20230518T191010
[2023-05-19T00:40:11.001+0530] {process_utils.py:79} INFO - Process psutil.Process(pid=10622, status='terminated', started='00:40:04') (10622) terminated with exit code None
[2023-05-19T00:40:11.016+0530] {process_utils.py:79} INFO - Process psutil.Process(pid=10484, status='terminated', exitcode=0, started='00:39:49') (10484) terminated with exit code 0
[2023-05-19T00:40:11.018+0530] {process_utils.py:79} INFO - Process psutil.Process(pid=10629, status='terminated', started='00:40:04') (10629) terminated with exit code None
